### DISCUSSION TITLE
Important!  Please read before making 1st posting...

### DISCUSSION THREAD

**Discussion by tech2serve** [code4recovery/12-step-meeting-list#412] [Announcements]
## 👋 Welcome!

We’re using Discussions as the primary place to provide TSML support. In most cases, you should _ask a question_ in the Q&A category, but the other categories are also open for community discussions!

Before asking a question, please search [our FAQ](https://github.com/code4recovery/12-step-meeting-list/wiki/Frequently-Asked-Questions) and past discussions as issues often get repeated.

When asking a question, please include details such as:

- The URL to your website.
- A description of what you are experiencing, and what you expected to have happen.
- Versions of TSML, Wordpress, etc.
- Screen caps to help us understand your concerns.
- Any errors you see on the website, and in particular through the browser's debugging/developer console (search the Internet for how to enable that feature).

It is also helpful if you perform some basic troubleshooting steps when you're seeing unexpected results. A number of one-off issues occur because of conflicts with other plugins or caches that get corrupted. Please disable other plugins and clear you website and browser caches to see if the issue resolves.

Discussions are a relatively new feature of GitHub, and there are still some features lacking. One thing you can do is to keep comments in the thread (as we cannot change them). This means that if another community member offers an answer to your question, please reply directly to that answer rather than starting another answer. This will help others to follow the thread.

If you receive a response that addresses your question, please go ahead and mark it as the answer! 

And if you wish, feel free to comment below with an introduction of yourself and tell us about what you do with our community!

**Comment by joshreisner** [comment]
👋 hi I'm Josh, I originally started the 12 Step Meeting List plugin. It first launched in May 2014 on the aasanjose.org website, and, a year later, I added it to the WordPress plugin directory. I continue to make pull requests here and there, but I'm grateful to the new primary maintainers for leading it into the future.

**Comment by brianw-area24** [comment]
Hi, I'm Brian W. I raised my hand a couple years ago when @tech2serve said he was looking for some people to help maintain the 12 Step Meeting List plugin. I was impressed with the work that Josh had done, and I wanted to help contribute to help keep the plugin going. I've submitted a few pull requests, and try to answer some support questions when I can. So grateful to be a part of the Code 4 Recovery team.
